frontend/src/app/core/interfaces/check-option.ts

```diff
@@ -0,0 +1,4 @@
+export interface ICheckOption {
```

Comment:
Could you please change the file name as well (this is related to all interfaces)?

Reply:
names changed to team-interface.tsm check-option-interface.ts, user-calendar-interface.ts